app/controllers/admins/users_controller.rb

```diff
@@ -49,7 +52,11 @@ def user
     end
 
     def last_super_admin_tries_to_update_its_role?
-      user.role_super_admin? && last_super_admin? && params[:user][:role] == User::USER_ROLES[:super_admin]
+      (user.role_super_admin? && last_super_admin? && (params[:user][:role] != User::USER_ROLES[:super_admin])) == true
```

Comment:
isn't "== true" redunant here?

Reply:
Thank you for your attention to my code.
You are right. It's a good idea to omit '== true' construction with external parentheses.
Everything works as it should.